src/server/config.rs

```diff
@@ -61,9 +65,13 @@ impl Default for Config {
             send_buffer_size: DEFAULT_SEND_BUFFER_SIZE,
             recv_buffer_size: DEFAULT_RECV_BUFFER_SIZE,
             end_point_concurrency: DEFAULT_END_POINT_CONCURRENCY,
+            end_point_txn_concurrency_on_busy: usize::default(),
+            end_point_small_txn_tasks_limit: DEFAULT_END_POINT_SMALL_TXN_TASKS_LIMIT,
             storage: StorageConfig::default(),
             raft_store: RaftStoreConfig::default(),
-        }
+        };
+        cfg.reset_end_point_txn_concurrency();
```

Comment:
very strange to adjust the endpoint in default function, why not using fixed default value directly?